component-classes/index.js

```diff
@@ -550,14 +550,14 @@ export const backwardsCompatibleClasses = {
 };
 
 export const pagination = {
+  button: 'hover:no-underline focus:no-underline focusable inline-flex justify-center items-center transition-colors ease-in-out min-h-[44px] min-w-[44px] p-4 rounded-full border-0 hover:bg-clip-padding',
   currentPage: 'block md:hidden p-8 font-bold',
-  icon: 'flex w-16 h-16 s-icon',
```

Comment:
These classes are used in other components, so need to add them to the `backwardsCompatibleClasses`.

Reply:
Oh I thought that it wasn't necessary since `flex`, `w-16` and `h-16` are being used in other classes, so thought that we would still build the css for them. 🙈